### ISSUE TITLE
Using Google login into website & setup the code for the login screen

### ISSUE THREAD

**Issue by mrossi113** [discapp/discapp#5]
I've found some documentation on how to integrate Google sign-in within web app. https://developers.google.com/identity/sign-in/web/sign-in

I will start working on this for we need to create a API console project and client ID. https://developers.google.com/identity/sign-in/web/devconsole-project

**Comment by mrossi113** [comment]
It all setup the API console project and client ID for a web application only.

I will need to update this once we have URL for where the button should be and the URL it will be redirected to once the user login to their Google account. 

Here is some code to use for the login screen: https://developers.google.com/identity/sign-in/web/

**Comment by cruoti** [comment]
This is awesome.  Thanks.

**Comment by mrossi113** [comment]
If we want a user to be able to sign out of the app we need to add this piece to html or jsp pages. 
```<a href="#" onclick="signOut();">Sign out</a>
<script>
  function signOut() {
    var auth2 = gapi.auth2.getAuthInstance();
    auth2.signOut().then(function () {
      console.log('User signed out.');
    });
  }
</script>
```

**Comment by cruoti** [comment]
Sign out?  Once they're in, they're in for good.  :)

**Comment by cruoti** [comment]
I get an error message every time I try to sign in.  

The screen shot is from when I run it locally which is why you see the URL as localhost:8080.  If I try to run it from the public URL from our Elastic Beanstalk then it that URL is listed instead.

![sign-in issues](https://user-images.githubusercontent.com/25518813/27253005-196bcdd8-5339-11e7-98a8-3acfef15e629.PNG)

**Comment by mrossi113** [comment]
@cruoti yeah I have not put the URLs into the Oauth client yet for the redirect and login screen for Elastic Beanstalk URLs. This needs to be in the authO Client before using the login. Also that means the URL the I put in are the only ones the login will work with. So that why I added the "Skip to Homepage" link for when we use localhost:8080 for you will see the same error as above even with URLs be added.
